furusiya/main.py

```diff
@@ -1,20 +1,84 @@
 #!/usr/bin/env python3
+import random
 from datetime import datetime
 
 from tcod import image_load
-import colors
 
-# Has to be here, because we use it everywhere
+import colors
+from constants import SCREEN_WIDTH, SCREEN_HEIGHT, MAP_WIDTH, MAP_HEIGHT, PANEL_HEIGHT, LIMIT_FPS
+from game import Game
+from model.helper_functions.message import message
 from model.config import file_watcher, config
-file_watcher.watch('config.json', lambda raw_json: config.load(raw_json))
+from model.entities.party.player import Player
+from model.entities.party.stallion import Stallion
+from model.key_binder import KeyBinder
+from model.maps import generators
+from model.maps.area_map import AreaMap
+from model.maps.generators import DungeonGenerator
+from model.systems.xp_system import XPSystem
+from view.adapter.tdl_adapter import TdlAdapter
+from view.data.saveload import SaveLoad
+from view.map_renderer import MapRenderer
 
-from function_mess import new_game, play_game, load_game
-from main_interface import Game
-from constants import SCREEN_WIDTH, SCREEN_HEIGHT
-import random
+
+def new_game():
+
+    Game.area_map = AreaMap(MAP_WIDTH, MAP_HEIGHT)

```

Comment:
Please add tests for all this logic as mentioned elsewhere. Default values, things not being `None`, etc. is fine -- don't need to test that the return value from other calls is sane.